res/sections/35-Routing.tex

```diff
@@ -1,7 +1,7 @@
 \chapter{Routing}
 
 Routing determine the ``good''\footnote{The minimum path typically} path (
-sequence of routers) thru network from source to destination.
```

Comment:
That's not a typo: https://www.wordreference.com/enit/thru

Reply:
Ok... so Aspell told me wrong.

Reply:
Yep sometimes you need to add some words as exceptions in `tools/dictionary.en.pws`